### ISSUE TITLE
Sentence reading in MS Word breaks after going into browse mode

### ISSUE THREAD

**Issue by andrew-l-d** [nvaccess/nvda#5911]
Reading by sentence in Word (alt-up and alt-down arrow) works until switching to browse mode.  When reverting from browse mode sentence reading does not work without restarting NVDA.  The issue has existed since at least 2015.3 (ran a portable copy to check).  Apologies if already reported - could not find anything by searching.

**Comment by jcsteh** [comment]
Duplicate of #5805.
